### ISSUE TITLE
Correct explanation of learning_rate

### ISSUE THREAD

**Issue by fritshermans** [INRIA/scikit-learn-mooc#798]
The [explanation](https://inria.github.io/scikit-learn-mooc/python_scripts/ensemble_hyperparameters.html#histogram-gradient-boosting-decision-trees) of the `learning_rate` in the HistGradientBoostingClassifier does not agree with the scikit-learn [documentation](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.HistGradientBoostingClassifier.html):

_"When fitting the residuals, we would like the tree to try to correct all possible errors or only a fraction of them. The learning-rate allows you to control this behaviour. A small learning-rate value would only correct the residuals of very few samples. If a large learning-rate is set (e.g., 1), we would fit the residuals of all samples."_

This explanation suggests that there is some selection of samples being made to which following trees make corrections. The documentation of scikit-learn says that the learning rate is a multiplicative factor, which I think is correct:

_"The learning rate, also known as shrinkage. This is used as a multiplicative factor for the leaves values. Use 1 for no shrinkage."_
